src/components/Article.jsx

```diff
@@ -1 +1,34 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const today = new Date();
+const dateISO = today.toISOString().slice(0, 10);
+const dateLocaleString = today.toLocaleDateString('en-US', {
+  year: 'numeric',
+  month: 'long',
+  day: 'numeric',
+});
```

Comment:
Move it inside your component Article and use variable 'date' instead of creating new date (just change today with date).

Reply:
Done)